pyproject.toml

```diff
@@ -18,3 +21,6 @@ test = [
   "pytest-xdist",
 ]
 doc = ["sphinx"]
+
+[tool.black]
+line-length = 79
```

Comment:
can we leave this at blacks default?

Reply:
+1

I know there's 1 knob in black and... I'd like us to stick with the default and avoid debating that. 🙈

Reply:
I hate Black’s 88 default and feel very strongly it’s interpreting the 80±10% rule wrong. But whatever, I have more important things to worry about.

While we’re on the topic of default values though, @pradyunsg can I move `tools/isort.cfg` to `.isort.cfg` (which is the default settings location)? I don’t use pre-commit, and the custom config location is messing with my manual isort calls :/

Reply:
Out of curiosity why not use pre-commit?

Reply:
It makes `git commit` extremely slow on Windows.

Reply:
Don't install the auto pre-commit feature? I usually call it manually as in ``tox -e fix_lint`` 😊

Reply:
> While we’re on the topic of default values though, @pradyunsg can I move `tools/isort.cfg` to `.isort.cfg` (which is the default settings location)?

Sure, go ahead and change that. :)